code/modules/surgery/organs/organ.dm

```diff
@@ -32,6 +32,13 @@ var/list/organ_cache = list()
 	var/sterile = 0 //can the organ be infected by germs?
 	var/tough = 0 //can organ be easily damaged?
 
+/obj/item/organ/Destroy()
+	processing_objects.Remove(src)
+	owner = null
```

Comment:
Suppose I could move that up the tree, yeah; would make sense. 

Not to self; put `processing_objects.Remove(src)` AFTER this, or else it won't truly be removed from processing.